lib/octopus.rb

```diff
@@ -18,10 +18,10 @@ def self.config
     @config ||= begin
       file_name = File.join(Octopus.directory, 'config/shards.yml').to_s
 
-      if File.exist?(file_name) || File.symlink?(file_name)
-        config ||= HashWithIndifferentAccess.new(YAML.load(ERB.new(File.read(file_name)).result))[Octopus.env]
+      config = if File.exist?(file_name) || File.symlink?(file_name)
+        HashWithIndifferentAccess.new(YAML.safe_load(ERB.new(File.read(file_name)).result))[Octopus.env]
```

Comment:
Would be nice to outsource it in another PR as this one isn't related to rails and would probably help to have it upstreamed faster.

Also, we should allow alisases here (or have a config for that)